### PR TITLE
add some style

### PR DESCRIPTION
**Pull request by johngruen**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,10 +1,112 @@
...
-<p>So, keep an eye on the universal-search mailing list; when we announce updates,
-you'll have to ensure you're authed (able to view this page), and then manually run
-the update (open about:addons, click gear icon at top right of page, then choose
-'Check for Updates' from the menu).
-<p>Have fun! Send your comments and thoughts to the universal-search list.
+<html>
+  <head>
+    <meta charset="utf-8">
+    
+    <title>Hello Frend</title>
```
Comment (pdehaan) [inline index.html:6]: Typo: "Friend"
Or is this some new slang with the kids that I'm not keen on?

File: index.html [hunk 2]
```diff
@@ -1,10 +1,112 @@
...
+      }
+
+      *, *:before, *:after {
+        box-sizing: inherit;
+        font-size: 12px;
+        font-weight: normal;
+        -webkit-user-select: none;
+           -moz-user-select: none;
+            -ms-user-select: none;
+                user-select: none;
```
Comment (pdehaan) [inline index.html:20]: Vendor prefixes even! :swoon:

**Comment by johngruen** [comment]
ALRIGHT, who let @pdehaan in here?

File: index.html (hunk 1 above)
Comment (johngruen) [inline index.html:6]: talk to @6a68

**Comment by jaredhirsch** [comment]
a veritable Masterpiece Theatre of GIFs. Bravura performance.

5/5 would merge again

**Pull request merged**